src/shell_minimal_runtime.html

```diff
@@ -4,14 +4,14 @@
 <script>
   var Module = {};
 
-  function script(url, cb) {
+  function s(url, cb) { // Downloads a script file and adds it to DOM
```

Comment:
did you minify these names manually because the minifier can't?

Reply:
Yeah, the minifier does not do minification on top level symbols, only inside functions when it knows there are no external references, so minified that manually.